peps/pep-0751.rst

```diff
@@ -0,0 +1,901 @@
+PEP: 751
+Title: A file format to list Python dependencies for installation reproducibility
+Author: Brett Cannon <brett@python.org>
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Created: 24-Jul-2024
+Replaces: 665
+
+Abstract
+========
+
+This PEP proposes a new file format for specifying what dependencies to
+(possibly) install into a Python environment for consistent installation
+repoducibility. The format is designed to be human-readable but
+machine-generated. Installers consuming the file should not need to resolve
+which listed dependencies to install based on other packages listed in the lock
+file, but instead evaluate each dependency in question in isolation.
```

Comment:
An attempt to avoid the term *lock file* before it is later defined, with the side benefit of phrasing the sentence in the positive mood:
```suggestion
machine-generated. Installers consuming the file should be able to
evaluate each package in question in isolation, with no need for
dependency resolution at install-time.
```